File: tst/server/ka.py
#!/usr/bin/python
from __future__ import print_function

import sys
import os


# what is CWD .. when scripts are called (!)

import cgi
import cgitb


print("Content-Type: text/plain", end="\r\n")
print("Connection: keep-alive", end="\r\n")
print("Content-Length: 17", end="\r\n\r\n")

# print ("I'm not dead yet.", end=""); # ugly
print ("I'm not dead yet.")

File: www/kd/other/ka.pl
#!/usr/bin/perl

use strict;
use warnings;
use CGI;

my $cgi = CGI->new;

print("Content-Type: text/plain\r\n");
# print("Connection: keep-alive\r\n");
# print("Content-Length: 11\r\n");
print("\r\n");
print("PERL is not dead (yet).");
